README.md

```diff
@@ -94,20 +94,6 @@ If you get a warning about the device's architecture, please follow the steps be
 Warning: Error creating LLDB target at path '*' using an empty LLDB target which can cause slow memory reads from remote devices: the specified architecture 'arm64-*-*' is not compatible with 'x86_64-apple-ios*' 
```

Comment:
The above warning about the device's architecture can also be removed since this answer addresses the issue mentioned.

Reply:
good catch. see f2a78b222f566e7bfc22ef1f95912f05d5301b6a